docs/attributes-registry/feature-flag.md

```diff
@@ -41,3 +42,19 @@ For example, the variant `red` maybe be used for the value `#c05543`.
 | `static` | The resolved value is static (no dynamic evaluation). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `targeting_match` | The resolved value was the result of a dynamic evaluation, such as a rule or specific user-targeting. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `unknown` | The reason for the resolved value could not be determined. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`feature_flag.evaluation.value_type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
`feature_flag.evaluation.value_type` makes sense, but I could also see `feature_flag.evaluation.value.type`, based on some of the other attribute keys. Is there some basic heuristic you applied in this case?

Reply:
I might actually like `feature_flag.evaluation.value.type` better because that leaves things like `feature_flag.evaluation.value.<size|format>` et cetera open for future consideration.